docs/src/pages/index.page.tsx

```diff
@@ -136,61 +136,80 @@ const HomePage = ({ colorMode, setThemeOverride, themeOverride }) => {
       lineHeight: '1.5',
     },
   };
+  const installScripts = {
+    react: `npm i @aws-amplify/ui-${framework} aws-amplify`,
+    vue: `npm i @aws-amplify/ui-${framework} aws-amplify`,
+    angular: `npm i @aws-amplify/ui-${framework} aws-amplify`,
+    flutter: 'flutter pub add amplify_authenticator',
+  };
+  const frameworkInstallScript = installScripts[framework];
   return (
     <>
-      <View as="section" className="docs-home-section-bg container">
+      <View as="section" className="container">
         <HomeLogo />
-        <Image
-          alt=""
-          className="docs-home-vue"
-          src="/svg/integrations/vue.svg"
-        />
-        <Image
-          alt=""
-          className="docs-home-react"
-          src="/svg/integrations/react.svg"
-        />
-        <Image
-          alt=""
-          className="docs-home-angular"
-          src="/svg/integrations/angular.svg"
-        />
-        <Card padding={tokens.space.xl} variation="outlined">
-          <Text fontSize={tokens.fontSizes.xl}>
-            Amplify UI is an open-source design system with cloud-connected
-            components and primitives that simplify building accessible,
-            performant, and beautiful applications in React, Angular, and Vue
-            (more coming soon).
-          </Text>
 
+        <Flex direction="row" padding={tokens.space.xl}>
+          <Card variation="outlined" flex="1">
+            <Text fontSize={tokens.fontSizes.xl}>
+              Amplify UI is an open-source design system with cloud-connected
+              components and primitives that simplify building accessible,
+              performant, and beautiful applications in React, Angular, Vue, and
+              Flutter (more coming soon).
+            </Text>
+
+            <Flex
+              direction={{ base: 'column-reverse', medium: 'row' }}
+              padding={`${tokens.space.medium} 0 0 0`}
+            >
+              <Button
+                size="large"
+                variation="primary"
+                as="a"
+                href={`/getting-started/installation?platform=${framework}`}
+              >
+                Get started
+                <IconChevronRight />
```

Comment:
Can we use the icon from `react-icons` package? We are planning on remove the built-in icon set in ui-react.

Reply:
`IconChevronRight` is used a few location in this file. It might make sense to replace them all at once when the set is removed from ui-react